### PR TITLE
Auto pass qc for non microsalt apptags

### PR DESCRIPTION
**Pull request by seallard**
## Description
Closes https://github.com/Clinical-Genomics/cg/issues/2820. 
According to @Karl-Svard, we should disable the microsalt qc for samples with apptags other than MWX/MWR/mic.

New requirement: as to ensure the virus samples have reads, a qc based on the percent guaranteed reads of the target reads will be applied according to @beatrizsavinhas and @karlnyr.

### Fixed
- Disable microsalt qc for non microsalt sample apptags

### This [version](https://semver.org/) is a
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: cg/meta/workflow/microsalt/quality_controller/quality_controller.py [hunk 1]
```diff
@@ -83,6 +84,9 @@ def quality_control_sample(self, sample_id: str, metrics: SampleMetrics) -> Samp
             and valid_10x_coverage
         )
 
+        if is_non_microsalt_apptag(sample):
```
Comment (seallard) [inline cg/meta/workflow/microsalt/quality_controller/quality_controller.py:87]: Is there a better name for such apptags?

Comment (diitaz93) [reply cg/meta/workflow/microsalt/quality_controller/quality_controller.py]: one solution is to create the opposite function, `has_sample_microsalt_apptag` and just check if that is False 😅

**Review by diitaz93 — APPROVED** [review]
👍

File: cg/meta/workflow/microsalt/quality_controller/quality_controller.py (hunk 1 above)
Comment (Karl-Svard) [reply cg/meta/workflow/microsalt/quality_controller/quality_controller.py]: you could name them "non_microbial" instead of "non_microsalt"

**Review by Karl-Svard — APPROVED** [review]
I haven't really been part of the microSALT QC project, but it looks good to me! 

FYI @beatrizsavinhas @karlnyr

**Pull request merged**